pkg/boot/fit/fit_test.go

```diff
@@ -25,11 +32,12 @@ func TestLoadConfig(t *testing.T) {
 	t.Logf("kernel name: %s", kn)
 	t.Logf("ramdisk name: %s", rn)
 	if kn != "kernel@0" {
-		t.Fatal(err)
+		t.Fatalf("Expected kernel %s, got %s", "kernel@0", kn)
```

Comment:
you can use Errorf here so if the next line is wrong too you'll get both.

Reply:
Wasn't thinking Errorf would also fail the test. Much better, thanks.